tests/jax/ops/test_comparison_operands.py

```diff
@@ -0,0 +1,77 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Callable
+
+import jax
+import jax.lax as jlx
+import jax.numpy as jnp
+import pytest
+from infra import run_op_test_with_random_inputs
+
+# NOTE TTNN does not support boolean data type, so bfloat16 is used instead.
+# Hence the output of comparison operation is bfloat16. JAX can not perform any
+# computation due to mismatch in output data type (in testing infrastructure).
+# The following tests explicitly convert data type of comparison operation
+# output for the verification purposes.
+# TODO Remove this workaround once the data type issue is resolved.
+# https://github.com/tenstorrent/tt-xla/issues/93
+
+# This should work in new infra due to match_dtypes CHECK TODO
```

Comment:
I'm pretty sure you can just remove it right away (all comments above and `convert_output_to_bfloat16` fixture), this should be solved with your infra changes.